plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/perspective/widgets/machine/appliance/server/ServerViewImpl.java

```diff
@@ -94,11 +94,11 @@ public String getValue(Server server) {
 
     /** {@inheritDoc} */
     @Override
-    public void setServers(@NotNull List<Server> servers) {
-        List<Server> list = new ArrayList<>();
-        for (Server server : servers) {
-           if (!Constants.TERMINAL_REFERENCE.equals(server.getRef())){//TODO: temporary hide terminal URL  
-               list.add(server);
+    public void setServers(@NotNull List<ServerEntity> servers) {
+        List<ServerEntity> list = new ArrayList<>();
+        for (ServerEntity serverEntity : servers) {
+           if (!Constants.TERMINAL_REFERENCE.equals(serverEntity.getRef())){//TODO: temporary hide terminal URL
```

Comment:
I'd say it's a bit hard to read such constructions, do you think it is possible to simplify that?